apps/flux-system/base/image-automation.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
+kind: ImageUpdateAutomation
+metadata:
+  name: imageautomation
+  namespace: flux-system
+spec:
+  interval: 1m0s
+  sourceRef:
+    kind: GitRepository
+    name: flux-config
+  git:
+    checkout:
+      ref:
+        branch: main
```

Comment:
Do we want to reference master until we switch to main?

Reply:
Good shout, missed that one.